guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Optional: Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
+
+You can set up your {Project} deployment to use an external PostgreSQL database during the installation of {Project}. 
+You can also migrate to an external PostgreSQL database on an existing {Project} deployment.
+
+.Procedure
+
+. Consider whether using an external database is beneficial for your use case. 
+For more information, see {AdministeringDocURL}postgresql-as-an-external-database-considerations_admin[PostgreSQL as an external database considerations] in _{AdministeringDocTitle}_.
```

Comment:
How about turning this into a prerequisite?

Reply:
Well, it would make the procedure somewhat shorter (though it's already pretty short, I think), but it might be a bit awkward information flow-wise. I'll give it a shot in the next iteration, nonetheless.